pkg/tf2pulumi/convert/eject_test.go

```diff
@@ -74,6 +78,50 @@ func isTruthy(s string) bool {
 	return s == "1" || strings.EqualFold(s, "true")
 }
 
+func formatTestFile(src io.Reader, dst io.Writer, isExperimental bool) error {
+	scanner := bufio.NewScanner(src)
+	inIf := false
+	inElse := false
+	for scanner.Scan() {
+		line := scanner.Text()
```

Comment:
It looks like calls to `scanner.Scan` will give back lines. Let's make sure we aren't effected by whitespace within the line here. 
```suggestion
		line := strings.TrimSpace(scanner.Text())
```